core/src/main/java/org/mule/runtime/core/internal/processor/strategy/AbstractProcessingStrategy.java

```diff
@@ -57,32 +55,39 @@ protected ExecutorService decorateScheduler(Scheduler scheduler) {
    */
   static final class ReactorSink implements Sink, Disposable {
 
-    private final BlockingSink<CoreEvent> blockingSink;
+    private final FluxSink<CoreEvent> fluxSink;
     private final reactor.core.Disposable disposable;
     private final Consumer onEventConsumer;
 
-    ReactorSink(BlockingSink<CoreEvent> blockingSink, reactor.core.Disposable disposable,
+    ReactorSink(FluxSink<CoreEvent> fluxSink, reactor.core.Disposable disposable,
                 Consumer<CoreEvent> onEventConsumer) {
-      this.blockingSink = blockingSink;
+      this.fluxSink = fluxSink;
       this.disposable = disposable;
       this.onEventConsumer = onEventConsumer;
     }
 
     @Override
     public void accept(CoreEvent event) {
       onEventConsumer.accept(event);
-      blockingSink.accept(event);
+      fluxSink.next(event);
     }
 
     @Override
     public boolean emit(CoreEvent event) {
       onEventConsumer.accept(event);
-      return blockingSink.emit(event) != BACKPRESSURED;
+      synchronized (fluxSink) {
+        if (fluxSink.requestedFromDownstream() > 0) {
```

Comment:
is this 0 a code or a quantity? if a code, is there a constant that we can use?

Reply:
quantity.